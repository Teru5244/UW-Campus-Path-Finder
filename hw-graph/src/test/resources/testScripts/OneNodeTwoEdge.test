# Create a graph
CreateGraph g

# Add one node
AddNode g n

# Add two edges
AddEdge g n n e1
AddEdge g n n e2

# Print the nodes in the graph and the outgoing edges from n
ListNodes g
ListChildren g n