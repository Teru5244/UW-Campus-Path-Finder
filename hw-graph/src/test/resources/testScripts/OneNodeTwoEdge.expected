# Create a graph
created graph g

# Add one node
added node n to g

# Add two edges
added edge e1 from n to n in g
added edge e2 from n to n in g

# Print the nodes in the graph and the outgoing edges from n
g contains: n
the children of n in g are: n(e1) n(e2)